projects/ngx-prx-styleguide/src/lib/image/image-loader.component.spec.ts

```diff
@@ -4,7 +4,8 @@ import { MockHalService } from '../hal/mock/mock-hal.service';
 import { DebugElement } from '@angular/core';
 import { ImageLoaderComponent } from './image-loader.component';
 
-describe('ImageLoaderComponent', () => {
+// TODO: convert to integration test
```

Comment:
The component's load event seems not to fire in Jest at all. I'm guessing that the load event is equivalent to setting the onload property on an [Image](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/Image).

If that's correct, then this is a good candidate for a test suite running in a real browser (like Cypress or Phantom). If not, maybe there's a way to get this to run in Jest that I'm missing.

Reply:
makes sense that it doesnt work here. Seems like a Cypress test candidate, but yeah like Ryan was saying not a huge deal.